### PR TITLE
Updated Readme with credits

### PR DESCRIPTION
**Pull request by mastermndio**
Ticket number

my task was to complete blah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,3 +128,6 @@ Incoming requests spawn a goroutine and are served concurrently, and the block c
 
 # Daemonize
 You can find examples of different daemon scripts for grimd on the [wiki](https://github.com/looterz/grimd/wiki/Daemon-Scripts).
+
+# Credits
+This entire project was written in a single day by the great one known a A-A-Ron
```
Comment (mastermndio) [inline README.md:133]: I really like how you did this all yourself you dapper young fellow

Comment (mastermndio) [reply README.md:133]: :-)

Comment (mastermndio) [reply README.md:133]: This was for educational purposes on Twitch

**Comment by looterz** [comment]
Checked out the stream, awesome class! great teacher 👍🏻

**Pull request closed without merging**